### DISCUSSION TITLE
updating dropbox files timestamps

### DISCUSSION THREAD

**Discussion by sgrizzi** [iterate-ch/cyberduck#16252] [Q&A]
I have been using Mountain Duck for some time connecting dropbox as a local folder, and uploading ad retrieving files with no problems.
Recently I realized that quite a few files in dropbox that I had uploaded some time ago, had been uploaded with a wrong timestamp.
Using dropbox mounted locally with Mountain Duck, I tried to re-sync the file timestamps using terminal command "rsync":
rsync -a -t --size-only /Volumes/BACKUP-source_directory  /Volumes/BACKUP/Destination_directory in dropbox.

The command works fine and, in finder, all my dropbox files show the correct updated timestamp. Yet, when I look at the files using dropbox web interface, the files timestamps appear untouched.
If I unmount dropbox using Mountain Duck and then remount dropbox, again, in finder, all files timestamp always appear updated, but the files in dropbox web interface still show the previous time stamp.
Mountain Duck appears "in synch", and dropbox is not showing and synch in process or pending.
Destination files in dropbox are "online-only".
Can anybody help me resolve this problem?

**Comment by dkocher** [comment]
The Dropbox API [^1] does not allow changing the modification date for existing files unfortunately.

[^1]: https://www.dropbox.com/developers/documentation/http/documentation

**Reply by sgrizzi** [reply]
Thanks for your very prompt reply.
Pity. I was trying to avoid re-uploading 45G of data but I see I have no alternative...
Best regards.
